src/App.js

```diff
@@ -5,51 +5,30 @@ import {
   Route,
   Link
 } from "react-router-dom";
+  import { ThemeProvider} from 'styled-components'
+import { HomePage, CoinPage, Portfolio } from './pages'
+import { darkTheme, lightTheme } from './components/Theme/Theme'
 
-export default function App() {
-  return (
-    <Router>
-      <div>
-        <nav>
-          <ul>
-            <li>
-              <Link to="/">Home</Link>
-            </li>
-            <li>
-              <Link to="/about">About</Link>
-            </li>
-            <li>
-              <Link to="/users">Users</Link>
-            </li>
-          </ul>
-        </nav>
 
-        {/* A <Switch> looks through its children <Route>s and
-            renders the first one that matches the current URL. */}
-        <Switch>
-          <Route path="/about">
-            <About />
-          </Route>
-          <Route path="/users">
-            <Users />
-          </Route>
-          <Route path="/">
-            <Home />
-          </Route>
-        </Switch>
-      </div>
-    </Router>
-  );
-}
 
-function Home() {
-  return <h2>Home</h2>;
-}
-
-function About() {
-  return <h2>About</h2>;
+export default class App extends React.Component {
+  state ={
+    theme: 'dark'
```

Comment:
what if you make the theme a boolean

Reply:
state = {
    dark: true,
  };
  render() {
    const theme = this.state.theme === true ? darkTheme : lightTheme;